addons/rating/controllers/main.py

```diff
@@ -24,15 +24,7 @@ def action_open_rating(self, token, rate, **kwargs):
         if rate not in (1, 3, 5):
             raise ValueError(_("Incorrect rating: should be 1, 3 or 5 (received %d)"), rate)
```

Comment:
We should get rid of this too
```suggestion
        # This route used to allow sending a rating with a GET, the
        # feature proved incompatible with various email providers and
        # has been removed.
```

Reply:
Along with the proper `# noqa:` to ignore the unused argument

Reply:
I am not super familiar with `# noqa:` but after a bit of research:
* variable prefixed with `_` are supposed to be ignored by recent version of ruff (as you stated previously)
* If I understood it correctly F841 was the old code for it (https://docs.astral.sh/ruff/rules/unused-variable/), but using it will create a warning to RUF100 (https://docs.astral.sh/ruff/rules/unused-noqa/), because it's an unused noqa and is not longer applicable

So should I still add it? Am I missing something?

Reply:
So removing this `if` statement is going to remove the only last place where the `rate` variable is used. I think ruff is going to complain about the variable being unused, and so red ci/style. I may be wrong. If ruff says it is "unused noqa" then we don't need to add it :)

In this very case we could also rename the variable `_rate`, but this is going to change the public API of the controller. Keep in mind that in controllers, all arguments can be passed via query-strings, i.e. it is *possible* (but *very* silly) to call this controller as `/rate/<access_token>/5?rate=3` (3 is overridden by 5), but renaming the variable you would get a warning for this URL "controller called ignoring {'rate'}".

Reply:
Actually `rate` is used when rendering the template. Please ignore my comment about `noqa` :sweat_smile:

Reply:
And also my comment about removing the `if` statement altogether...

Reply:
Okay, indeed seems like we were both confused ;)
Added the requested comment, so unless we need a second opinion we can go for an r+?
And then I will have fun with fixing the FW ports ;P

Reply:
> Okay, indeed seems like we were both confused ;) Added the requested comment, so unless we need a second opinion we can go for an r+? And then I will have fun with fixing the FW ports ;P

Would be interesting to test. Would be great to make a followup task in SM and ask for testing ... avoid submarine dev.